src/inmanta/data/model.py

```diff
@@ -411,8 +411,8 @@ def _count_all_resources(cls, summary_by_state: dict[str, int]) -> int:
 class LogLine(BaseModel):
     # Override the setting from the BaseModel class as such that the level field is
     # serialized using the name of the enum instead of its value. This is required
-    # to make sure that data sent to the API endpoints resource_action_update
-    # and send_deploy_done are serialized consistently using the name of the enum.
+    # to make sure that data sent to the send_deploy_done API endpoint
```

Comment:
What's this one still used for? I'd assumed it would go together with `resource_action_update`.